packages/vscode-extension/src/debug/prerequisitesHandler.ts

```diff
@@ -157,6 +175,12 @@ export async function checkAndInstall(): Promise<Result<any, FxError>> {
       // ignore telemetry error
     }
 
+    // terminate all running teamsfx tasks
+    if (allRunningTeamsfxTasks.size > 0) {
+      VsCodeLogInstance.info("Terminate all running teamsfx tasks.");
+      terminateAllRunningTeamsfxTasks();
```

Comment:
I remember we have a setting `fx-extension.stopTeamsToolkitTasksPostDebug`. Any side effect if always terminating all running tasks? Or, what if removing `fx-extension.stopTeamsToolkitTasksPostDebug`?

Reply:
Good catch here. I prefer removing this setting due to:
- The only (?) scenario is that user starts services manually then debug the attach configurations. Since we already provide the way to start services via npm scripts, user has no need to worry about the services being terminated after debug.
- This setting introduces difficulty for us to distinguish the running tasks, i.e. whether the tasks are held after last successful debug or not terminated as expected after last unsuccessful debug.
- This setting introduces difficulty for us to check existing ports. Actually current checking ports logic do not consider this setting...

How do you think about?

Reply:
Make sense to me. Agree to removing the setting.

Reply:
Sure, removed.